x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/add_source/add_custom_source/save_custom.tsx

```diff
@@ -104,13 +105,13 @@ export const SaveCustom: React.FC = () => {
                 heading="h3"
                 size="s"
                 title={
-                  <EuiLinkTo target="_blank" to={'https://www.elastic.co/kibana/feedback'}>
+                  <EuiLink href={'https://www.elastic.co/kibana/feedback'} external>
```

Comment:
No need for a string literal here
```suggestion
                  <EuiLink href="https://www.elastic.co/kibana/feedback" external>
```